app/models/spotlight/resources/oaipmh_harvester.rb

```diff
@@ -5,7 +5,15 @@
 module Spotlight::Resources
   class OaipmhHarvester < Spotlight::Resource
     attr_accessor :set, :base_url, :mapping_file
-    self.document_builder_class = Spotlight::Resources::OaipmhBuilder
+    def self.indexing_pipeline
+      @indexing_pipeline ||= super.dup.tap do |pipeline|
+        # if, say, you wanted to feed the transform with multiple source documents (here, by calling the `#iiif_manifest` method on the DlmeJson instance); previously, the #to_solr method of the document builder would have done this extraction
+        # pipeline.sources = [Spotlight::Etl::Sources::SourceMethodSource(:iiif_manifests)]
+        pipeline.transforms = [
+          ->(data, p) { OaipmhBuilder.new(p.context.resource).to_solr }
```

Comment:
I don't think this line is correct; it's still trying to index the OAI records as the harvester's own data

Reply:
This line is fine, it is the to_solr method itself that is the issue. I fixed it but changed it back based on what Rob needed. I think we should hold off on changing this until we have a harvester page that we can run re-imports from. otherwise there is no way to update from the UI.